### ISSUE TITLE
Recording shows wrong mouse cursor icon

### ISSUE THREAD

**Issue by fbruetting** [phw/peek#313]
As you see in my screen cast, inside the header bars, Peek shows wrong mouse cursors. In reality, I see the correct standard “pointer” icon, it’s just Peek showing that “window size scaling” icons.

The screen cast was done for showing the mouse cursor theme not being applied by Geary, so the different icons you see are not a problem of Peek,

[peek_geary.zip](https://github.com/phw/peek/files/1921859/peek_geary.zip)

**Comment by phw** [comment]
This is on gnome, right? If so, is this reproducible with the integrated Gnome Shell screen recording?

What happens if you launch peek with `peek -b ffmpeg`?

**Comment by fbruetting** [comment]
No, there are other errors occurring with Gnome’s internal recording function (resizing icons aren’t shown at all for both cursor themes).

FFMPEG doesn’t work at all, as you see in the attachment. Maybe because of UHD and Wayland?

[Peek 2018-04-30 20-44.zip](https://github.com/phw/peek/files/1961804/Peek.2018-04-30.20-44.zip)

Peek by the way shows `Gtk-Message: Failed to load module "pk-gtk-module"` in the command line. And I run it via Flatpak.
